statement.go

```diff
@@ -1,56 +1,52 @@
 package duckdb
 
-/*
-#include <duckdb.h>
-*/
-import "C"
-
 import (
 	"context"
 	"database/sql/driver"
 	"errors"
 	"fmt"
 	"math/big"
-	"unsafe"
+
+	m "github.com/marcboeker/go-duckdb/mapping"
 )
 
-type StmtType C.duckdb_statement_type
+type StmtType m.StatementType
```

Comment:
nit: I would call it `StatementType`

Reply:
We export this type, so doing that would break backward compatibility. I could add a type alias. Also, we use `Stmt` in other places, so this name is more consistent with our `Stmt` type. Personally, I'd keep it as-is, but your call! :)